src/models2/SubredditRule.es6.js

```diff
@@ -0,0 +1,34 @@
+import RedditModel from './RedditModel';
+import { SUBREDDIT_RULE } from './thingTypes';
+
+const T = RedditModel.Types;
+
+export default class SubredditRule extends RedditModel {
+  static type = SUBREDDIT_RULE;
+
+  static PROPERTIES = {
+    createdUTC: T.number,
+    description: T.string,
+    descriptionHTML: T.string,
+    kind: T.string,
+    priority: T.number,
+    shortName: T.string,
```

Comment:
Is that a thing in this context? (these aren't react proptype validators, they are [these things](https://github.com/reddit/node-api-client/blob/3X/src/apiBase/Model.es6.js#L37-L61)